src/main/java/org/jabref/logic/importer/fetcher/IssnFetcher.java

```diff
@@ -37,7 +37,7 @@ public IssnFetcher(ImportFormatPreferences importFormatPreferences) {
     public Optional<BibEntry> performSearchById(String identifier) throws FetcherException {
 
         Optional<String> checkedId = issnChecker.checkValue(identifier);
-        if (checkedId.isPresent()) {
+        if (checkedId.equals(Optional.empty())) {
```

Comment:
The opposite of Optional.isPresent is Optional.isEmpty

Reply:
Yes, I solved it incorrectly, I should change the ISSN Checker logic because when a number has a valid checksum, it's returning an optional empty. The logic should be when a number doesn't have a valid checksum, it should return an "incorrect control digit".